src/guide/jbs-jdk-bug-system.md

```diff
@@ -214,6 +214,7 @@ When triaging an issue, first give it a general review.
 
 1. If the issue is a duplicate, close it as such.
 1. If the issue belongs to a different area (it was filed in libraries, but it's an HotSpot issue), transfer it to the correct component/subcomponent making sure that the state remains [New]{.jbs-value}.
+   * Please note that issues in the [hotspot]{.jbs-value} component must have a [Subcomponent]{.jbs-field} set as well.
```

Comment:
Per me, this is true for `security-libs` component also. Where unsure, we use `java.security` or` jdk.security` sub-components but a sub-component must be set. In addition, can we request to remove `Empty` sub-component from the list in JBS if it is possible to do so?